packages/sdk/src/swap/sdk.ts

```diff
@@ -102,6 +111,12 @@ export class SwapSDK {
     return this.supportedAssets;
   }
 
+  private async getBoostPoolsDepth(): Promise<BoostPoolsDepth> {
+    this.boostPoolsDepth ??= await getAllBoostPoolsDepth(this.rpcConfig);
+
+    return this.boostPoolsDepth;
```

Comment:
if we memoize this the data will get stale quickly
```suggestion
    return getAllBoostPoolsDepth(this.rpcConfig);
```

Reply:
Done